scripts/orchestration/qoder_dispatch_bridge.py

```
@@ -0,0 +1,598 @@
+
+REPO_ROOT = Path(__file__).resolve().parents[2]
+
+# ---------------------------------------------------------------------------
+# Optional imports with graceful fallback
+# ---------------------------------------------------------------------------
+
+_routing_graph: Optional[Dict[str, Any]] = None
+_routing_loader_available = False
+
+try:
+    from scripts.orchestration.routing_graph_loader import (
```

Flake8 (7.3.0)

[error] 'scripts.orchestration.routing_graph_loader.DomainRoute' imported but unused

(F401)